t/producer.t

```diff
@@ -294,3 +298,38 @@ GET /t
 offset: 2000LL
 --- no_error_log
 [error]
+
+=== TEST 8: sasl simple send
```

Comment:
need three blank lines before this line, you are recommended to use `reindex` to format this file automatically.
https://github.com/openresty/openresty-devel-utils#reindex